builtin/plugins/dposv3/dpos.go

```diff
@@ -1349,19 +1354,42 @@ func ShiftDowntimeWindow(ctx contract.Context, currentHeight int64, candidates [
 	if state.Params.DowntimePeriod != 0 && (uint64(currentHeight)%state.Params.DowntimePeriod) == 0 {
 		ctx.Logger().Info("DPOS ShiftDowntimeWindow", "block", currentHeight)
 
+		downtimeSlashingEnabled := ctx.FeatureEnabled(loomchain.DPOSVersion3_4, false)
+		maximumMissedBlocksBig := CalculateFraction(state.Params.MaxDowntimePercentage.Value, loom.BigUInt{big.NewInt(int64(state.Params.DowntimePeriod))})
```

Comment:
Once this feature has been deployed, `state.Params.MaxDowntimePercentage` is going to be nil. So we need check it before dereferencing it.